src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/_help.py

```diff
@@ -26,6 +26,13 @@
 helps['redis update-settings'] = """
     type: command
     short-summary: Update the settings of a redis cache.
+    long-summary: |
+        WARNING: This command is being deprecated. Please use 'update' command
```

Comment:
Recommend you add (DEPRECATED) to the short summary so it shows up in `az redis -h`. I like that the long summary directs the user to the right command.